non-wheel/dbtoxlsx/README.md

```diff
@@ -0,0 +1,21 @@
+Sample usage:
```

Comment:
Could you add the "pip3 install xlsxwriter" instructions?
Also, to get the script to run, I had to run "python3 dbtoxlsx.py"